src/data/int/basic.lean

```diff
@@ -447,6 +447,12 @@ end
 @[simp] theorem mod_mod (a b : ℤ) : a % b % b = a % b :=
 by conv {to_rhs, rw [← mod_add_div a b, add_mul_mod_self_left]}
 
+lemma sub_mod (a b n : ℤ) : (a - b) % n = ((a % n) - (b % n)) % n :=
```

Comment:
Does `neg_mod` already exists? Otherwise, I envision the proof would be
```
rw [sub_eq_add_neg, add_mod, neg_mod, sub_eq_add_neg]
```

Reply:
No `neg_mod` to be found. But also I think your proof assumes it would be `(-a) % b = - (a % b)`, but this isn't true.

Reply:
Hmm, right, this statement quickly becomes tricky. And you have a proof, so let's not bother.